packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -79,15 +82,38 @@ export default class LinkCore extends React.Component<Props> {
             <Icon
                 icon={externalIconPath}
                 size="small"
-                style={iconStyles.icon}
+                style={iconStyles.endIcon}
                 testId="external-icon"
             />
         );
 
+        const startIconChild = startIcon ? (
+            <Icon
+                icon={startIcon}
+                size="small"
+                style={iconStyles.startIcon}
+                testId="start-icon"
+            />
+        ) : (
+            <></>
+        );
+
+        const endIconChild = endIcon ? (
+            <Icon
+                icon={endIcon}
+                size="small"
+                style={iconStyles.endIcon}
+                testId="end-icon"
+            />
+        ) : (
+            target === "_blank" && externalIcon
+        );
+
         const linkContent = (
             <>
-                {children}
-                {target === "_blank" && externalIcon}
+                {startIconChild}
```

Comment:
I think you can remove the ternary operator for `startIconChild` definition and only set it to just the `<Icon>`, and then down here, use `{startIcon && startIconChild}`.